### PR TITLE
Add VertexID, SrcID, DstID, Rank in FETCH query result

### PR DESCRIPTION
**Pull request by laura-ding**

### CODE REVIEW

File: src/graph/test/DataTest.cpp [hunk 1]
```diff
@@ -257,14 +257,14 @@ TEST_F(DataTest, InsertTest) {
...
+        std::vector<std::tuple<int64_t, std::string, int64_t>> expected = {
+                {std::hash<std::string>()("Bob"), "Bob", 9},
         };
         ASSERT_TRUE(verifyResult(resp, expected));
         cmd = "FETCH PROP ON student hash(\"Bob\")";
         code = client_->execute(cmd, resp);
-        std::vector<std::tuple<std::string, int64_t>> expected2 = {
-                {"four", 20191106001},
+        std::vector<std::tuple<int64_t, std::string, int64_t>> expected2 = {
+                {std::hash<std::string>()("Bob"), "four", 20191106001},
```
Comment (yixinglu) [inline src/graph/test/DataTest.cpp]: hash("Bob") ? Isn't it `hash("four")` ?

Comment (laura-ding) [reply src/graph/test/DataTest.cpp]: The first column is the vertexID

File: src/graph/FetchEdgesExecutor.cpp [hunk 2]
```diff
@@ -49,6 +49,11 @@ Status FetchEdgesExecutor::prepareClauses() {
         if (!status.ok()) {
             break;
         }
+
+        // Add SrcID, DstID, Rank before prepareYield()
+        returnColNames_.emplace_back("SrcID");
+        returnColNames_.emplace_back("DstID");
+        returnColNames_.emplace_back("Rank");
```
Comment (CPWstatic) [inline src/graph/FetchEdgesExecutor.cpp]: I suggest we use the reserved property name, `_src`, `_dst`, `_rank`

File: src/graph/FetchVerticesExecutor.cpp [hunk 3]
```diff
@@ -57,6 +57,9 @@ Status FetchVerticesExecutor::prepareClauses() {
             LOG(ERROR) << "Prepare vertex id failed: " << status;
             break;
         }
+
+        // Add VertexID before prepareYield()
+        returnColNames_.emplace_back("VertexID");
```
Comment (CPWstatic) [inline src/graph/FetchVerticesExecutor.cpp]: `_id`

File: src/graph/FetchEdgesExecutor.cpp (hunk 2 above)
Comment (laura-ding) [reply src/graph/FetchEdgesExecutor.cpp]: `_src` and ` _dst` and `_rank` are internal names

**Comment by laura-ding** [comment]
@Amber1990Zhang here need to update docs about `FETCH`
image
image
Add some columns in result about `FETCH`

File: src/graph/FetchEdgesExecutor.cpp (hunk 2 above)
Comment (CPWstatic) [reply src/graph/FetchEdgesExecutor.cpp]: It is, but we would better not rename for users. And users could also use the internal names in the nGQL. By the way, the storage has considered this situation, and return these value by default, please check it.

Comment (CPWstatic) [reply src/graph/FetchEdgesExecutor.cpp]: Emm, `_src`, `_dst`, `_rank`, they are not internal names, i think. They are just reserved prop name which could be used to get the `key` of a vertex/edge. For example, people could use `<edge>._src` to get a source vertex id in an edge.

**Pull request merged**